### DISCUSSION TITLE
the link in the text doesn't properly show

### DISCUSSION THREAD

**Discussion by miraculusik** [nvim-neorg/neorg#753] [Q&A] [answered]
<img width="605" alt="Screenshot 2023-02-18 at 01 41 12" src="https://user-images.githubusercontent.com/66497631/219811827-e4e6b811-d980-42ae-983a-e787ea4873db.png">

As the shows in image, words in the brackets doesn't disappear. How can i fix it? btw links works properly just shows wrong

**Comment by max397574** [comment, marked as answer]
:h conceallevel

**Reply by miraculusik** [reply]
it looks like the problem is caused by this 
conceallevel default value is 0. how can i change it?
this is my neorg setup
```lua
require("neorg").setup({
	load = {
		["core.defaults"] = {},
		["core.norg.completion"] = { config = { engine = "nvim-cmp" } },
		["core.norg.concealer"] = {
			config = {
				icon_preset = "varied",
				folds = false,
				dim_code_blocks = {
					conceal = true,
					enabled = false,
					padding = { left = 2 },
				},
			},
		},
		["core.norg.dirman"] = {
			config = {
				workspaces = {
					notes = "~/notes",
				},
			},
		},
	},
})

```

**Reply by max397574** [reply]
this is just a normal neovim option
`vim.opt.conceallevel=2` (or whatever you want)

**Reply by miraculusik** [reply]
it works thank you :)
